src/components/interface/settings/SettingsDropdown.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import SettingsIcon from '@material-ui/icons/Settings';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import { auth } from '../../../firebase/firebase';
+import styles from '../../styles';
+
+class SettingsDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      anchorEl: undefined
+    };
+  }
+
+  handleClick = (event) => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: undefined });
+  };
+
+  handleLogout = () => {
+    auth.signOut();
+  };
+
+  render() {
+    const { anchorEl } = this.state;
+    const { classes } = this.props;
+    return (
+      <div className={classes.settingsDropdown_wrapper}>
+        <SettingsIcon
+          aria-owns={anchorEl && 'simple-menu'}
+          aria-haspopup="true"
+          onClick={this.handleClick}
+          style={{ cursor: 'pointer' }} />
+        <Menu
+          id="simple-menu"
+          anchorEl={anchorEl}
+          open={Boolean(anchorEl)}
+          onClose={this.handleClose}>
+          <Link to="/settings">
+            <MenuItem onClick={this.handleClose}>Settings</MenuItem>
```

Comment:
For the dropdown, I think Settings is wrapped with a `link` while Logout isn't causing it to have different styling: i.e. 
![image](https://user-images.githubusercontent.com/18960825/51093426-2eb9cc00-1758-11e9-804a-05299e3b103d.png)

if you add 
```
a:focus {
  outline: none;
}
```
to the `index.css` it should fix the styling inconsistency

Reply:
This is resolved in the latest push.